app/styles/layout/_feeds.scss

```diff
@@ -120,7 +120,8 @@
   width: 100%;
   position: relative;
   @media (max-width: 768px) {
-    margin: 0;
+    margin-left: 8px;
+    margin-right: 8px;
```

Comment:
I'm not a fan of this change. At least not on smaller phones. It looks good on my OnePlus 6T but on my iPhone X, it takes up too much space. I suggest you add another media query for smaller screens.

```css
@media (max-width: 380px) {
    margin-left: 2px;
    margin-right: 2px;
}
```

This also applies to the full-page posts.